mantis-publish/mantis-publish-core/build.gradle

```diff
@@ -17,7 +17,7 @@ ext {
     archaiusVersion = 'latest.release'
     versionMockito = '2.18.+'
     mqlVersion = '3.2.2'
-    spectatorVersion = '0.134.0'
```

Comment:
Can we get rid of the version here and instead use ext.libraries in the root build.gradle?